src/utils/cmakeUtil.mts

```diff
@@ -238,9 +241,8 @@ export async function cmakeUpdateSDK(
 ): Promise<boolean> {
   // TODO: support for scaning for seperate locations of the CMakeLists.txt file in the project
   const cmakeFilePath = join(folder.fsPath, "CMakeLists.txt");
-  const sdkPathRegex = /^set\(PICO_SDK_PATH\s+([^)]+)\)$/m;
-  const toolchainPathRegex = /^set\(PICO_TOOLCHAIN_PATH\s+([^)]+)\)$/m;
-  const toolsPathRegex = /^set\(pico-sdk-tools_DIR\s+([^)]+)\)$/m;
+  const updateSectionRegex =
```

Comment:
Maybe a short comment indicating that this regex requires the multiline (m) and the dotall (s) feature to be enabled. Just for testing this regex in external tools and if maybe the nodejs regex defaults for this flags should change.